webb_ai/README.md

```diff
@@ -0,0 +1,55 @@
+# Webb.ai
+
+## Overview
+
+Matt by Webb.ai is the first AI-enabled reliability engineer.
+Matt troubleshoots monitors and alerts from Datadog, Kubernetes, and cloud providers like AWS, Azure, and GCP.
+It identifies the root cause of an alert or incident in less than 5 mins.
+
+With this integration, Matt automatically identifies alerts from Datadog and troubleshoots them, and the mean time to debug those alerts is reduced significantly.
+
+This integration queries the following data:
+- Datadog events
+- Datadog metrics and tags
+
+This integration sends events to Datadog, including the root cause analysis performed by Matt and changes in Kubernetes clusters.
+You can view detailed RCA analysis for your alerts, including all hypotheses analyzed and the exact steps Matt took with supporting evidence.
+With this analysis, 80-90% of troubleshooting is automated, greatly reducing the debugging time of on-call engineers. 
```

Comment:
```suggestion
With this integration, events are sent to Datadog including the root cause analysis performed by Matt and changes in Kubernetes clusters. The events include detailed RCA analysis for your alerts such as the steps Matt took and all analyzed hypotheses to reduce debugging time of on-call engineers. 
```

Reply:
This edit changes the meaning of the prose and I don't think it's correct.